test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -15,6 +15,7 @@
 using Microsoft.ML.Trainers.FastTree;
 using Microsoft.ML.Transforms;
 using Xunit;
+using Microsoft.ML.Calibrators;
```

Comment:
Can you sort the usings?

Reply:
Put the `Microsoft.ML.Calibrators` in the correct spot in the list.